src/annotator/index.js

```diff
@@ -53,15 +53,14 @@ function init() {
 
     const hypothesisAppsOrigin = new URL(sidebarConfig.sidebarAppUrl).origin;
     portProvider = new PortProvider(hypothesisAppsOrigin);
+    portProvider.listen();
```

Comment:
If the portProvider starts listening at this point, there is a risk that a `Guest` able to discover the host and triggers the `frameConnected` event before the `Sidebar` is created.

Reply:
Any messages posted to the host frame won't be handled until the next turn of the event loop. With this change we are still registering the listener in the same event loop turn, just earlier.

Reply:
OK, then I will approve the PR.